include/picongpu/particles/atomicPhysics/kernel/CheckForOverSubscription.kernel

```diff
@@ -124,13 +124,13 @@ namespace picongpu::particles::atomicPhysics::kernel
                  &rejectionProbabilityCacheCell,
                  &sharedResourcesOverSubscribed](uint32_t const linearCellIndex)
                 {
-                    CalculateRejectionProbability::ofCell(
-                        linearCellIndex,
-                        superCellCellOffset,
-                        eFieldBox,
-                        eFieldEnergyUseCache,
-                        rejectionProbabilityCacheCell,
-                        sharedResourcesOverSubscribed);
+                    if(CalculateRejectionProbability::ofCell(
```

Comment:
^^
this will do the same
```C++
sharedResourcesOverSubscribed=CalculateRejectionProbability::ofCell(
                           linearCellIndex,
                           superCellCellOffset,
                           eFieldBox,
                           eFieldEnergyUseCache,
                           rejectionProbabilityCacheCell));
```

Reply:
No that is not equivalent.

We want only ever want to set the flag `sharedRessourcesOverSubscribed` to `true` never to `false` since setting it to false would overwrite any previous setting to true and this vairable is supposed to track if there was at least one cell that was oversubscribed.

As discussed offline, using `||=` (or with previous value) would be an alternative but we will keep the current version for now.